batchspawner/api.py

```diff
@@ -1,6 +1,7 @@
 import json
-from tornado import web
+
 from jupyterhub.apihandlers import APIHandler, default_handlers
+from tornado import web
```

Comment:
A [recommended](https://peps.python.org/pep-0008/#imports) order of imports is

standard library:
> `import json`

then related third part imports:
> `from tornado import web`

then local application/library specific imports:
> `from jupyterhub.apihandlers import APIHandler, default_handlers`

Although they also recommend putting a blank line in between each group, so one line before tornado. I suppose this change would be congruent with the recommendation if tornado is categorized as local application rather than related third party, but I think of it as more the latter.

Reply:
The formatting is a result of autoformatting via pre-commit that has declared use of `isort`, configured with `--profile=black`.

![image](https://github.com/jupyterhub/batchspawner/assets/3837114/28fb104e-c52a-413d-a5da-a4c05f6a654f)

I figure the behavior now is that `tornado` and `jupyterhub` are considered third party imports and grouped together.

I think [isort can be configured to associate specific imports in different ways](https://pycqa.github.io/isort/docs/configuration/options.html#known-local-folder), for example considering jupyterhub or tornado as something to be put in another category.

I'm very positively towards the use of autoformatting, but I have no big opinions on how.

Reply:
I read about third party and first party imports in https://github.com/PyCQA/isort/issues/1057.

I think for example pytest, tornado, and jupyterhub make sense to see as third party imports, while `import batchspawner` would be a first party import, and `import .batchspawner` would be a local import.